### ISSUE TITLE
Downloading Exercise folders (Setup and Command Line) does not work

### ISSUE THREAD

**Issue by konradpa** [lnnrtwttkhn/version-control-book#236]
On the new "exercises" site, I can only properly download the "First Steps with Git and the "Branches" exercise repository.

Downloading the "Setup" or "Command Line" repo/folder just gives me an empty `recipe.txt`file

**Comment by lnnrtwttkhn** [comment]
yup, the empty text file is created in the CLI exercise and the setup exercise is only Git configuration.

But is it still the case for you that the zip file only includes the file but not the folder?

**Comment by konradpa** [comment]
> yup, the empty text file is created in the CLI exercise and the setup exercise is only Git configuration.
> 
> But is it still the case for you that the zip file only includes the file but not the folder?

Yes, It only downloads the empty .txt file without any folder on my end

**Comment by lnnrtwttkhn** [comment]
Hmm that's weird. Could you check if this is a setting of your zip command on your computer?
